tests/mock/MockNetworkListener.h

```diff
@@ -59,7 +59,7 @@ class MockNetworkListener : public QTcpServer
     DEFLECT_API virtual ~MockNetworkListener();
 
 protected:
-    void incomingConnection(int handle) override;
+    void incomingConnection(int handle);
```

Comment:
Oh, I missed this file. My bad. I slightly remember the 'final' problem, but so far it works...